build/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.5
+
+MAINTAINER Peter Mattis <peter@cockroachlabs.com>
+
+RUN curl --silent --location https://deb.nodesource.com/setup_4.x | bash - && \
+ apt-get dist-upgrade -y && \
+ apt-get install --no-install-recommends --auto-remove -y git build-essential file nodejs && \
+ apt-get clean autoclean && \
+ apt-get autoremove -y && \
+ rm -rf /tmp/*
+
+COPY bower.json npm-shrinkwrap.json package.json tsd.json Makefile ./
+
+RUN make bower.installed npm.installed tsd.installed && \
+ rm bower.json npm-shrinkwrap.json package.json tsd.json Makefile && \
```

Comment:
Heh, who's going to read that message? This will already do the right thing (download new deps) if they get updated, and if we see a problem (like the github API limit failures) then we'll know to update the image.